src/main/java/core/basesyntax/service/impl/ConvertWriteDataServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ConvertWriteDataService;
+import java.util.Map;
+
+public class ConvertWriteDataServiceImpl implements ConvertWriteDataService {
+    private static final String TITLE_STRING = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public String convertDataToFile(Map<String, Integer> fruitStorage) {
```

Comment:
```suggestion
    public String getReport() {
```
You should use your dao to access storage. Make Dao field in this class.